CONTRIBUTING.md

```diff
@@ -258,3 +258,18 @@ the main branch. The workflow file should be named as
 [![NuGet version badge](https://img.shields.io/nuget/v/{your_package_name})](https://www.nuget.org/packages/{your_package_name})
 [![NuGet download count badge](https://img.shields.io/nuget/dt/{your_package_name})](https://www.nuget.org/packages/{your_package_name})
 ```
+
+### Guidance for target frameworks
+
+Components shipped from this repo SHOULD comply with the following requirements:
```

Comment:
If `SHOULD` is used here, the bullet points don't need to and shouldn't have MUST/SHOULD anymore.